### PR TITLE
fixed input on MacOSX

### PR DESCRIPTION
**Pull request by PaperPrototype**
Screenshot 2024-03-30 at 3 13 01 PM

### CODE REVIEW

**Comment by PaperPrototype** [comment]
File read write is broken though

**Comment by PaperPrototype** [comment]
I think the window's style slash -> `/assets\Defaults` is where the issue might be

**Comment by michaelsakharov** [comment]
Is the ImGui.ini file needed?

**Comment by PaperPrototype** [comment]
no

File: Prowl.Editor/ImGUI/ImGUIController.cs [hunk 1]
```diff
@@ -237,15 +255,17 @@ private void UpdateImGuiInput()
 
             var keyState = _input.Keyboards[0];
 
-            io.MouseDown[0] = _input.Mice[0].IsButtonPressed(MouseButton.Left);
```
Comment (PaperPrototype) [inline Prowl.Editor/ImGUI/ImGUIController.cs:240]: this is where the code actually got changed

**Pull request merged**